plugins/wasm-cpp/extensions/basic_auth/README_EN.md

```diff
@@ -0,0 +1,117 @@
+<p>
+   English | <a href="README.md">中文</a>
+</p>
+
+# Description 
+`basic-auth` plugin implements the function of authorization based on the HTTP Basic Auth standard.
```

Comment:
"Auth" in basic-auth should stand for "authentication" instead of "authorization".

https://en.wikipedia.org/wiki/Basic_access_authentication